huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go

```diff
@@ -84,11 +84,13 @@ func ResourceDmsKafkaInstance() *schema.Resource {
 				ForceNew: true,
 			},
 			"availability_zones": {
-				Type:     schema.TypeList,
+				// There is a problem with order of elements in Availability Zone list returned by Kafka API.
+				Type:     schema.TypeSet,
```

Comment:
just a question: do we need to update the type to Set on docs. I just find we only have one `Set` type filed on `compute_servergroup`, not sure if it's needed.

Reply:
This question is actually about whether the user refers to this field, because the processing of the two types of Set and List is different.
For the breaking change declear,  we need to make it clear.
```
~> After 1.34.1: The parameter behavior of `availability_zones` has been changed from `TypeList` to `TypeSet`.
```